packages/beacon-node/test/spec/presets/ssz_static.test.ts

```diff
@@ -58,24 +58,22 @@ const sszStatic =
       (ssz.phase0 as Types)[typeName];
 
     if (!sszType) {
-      expect.fail(
-        `Missing SSZ type definition for ${typeName}; this will prevent associated ssz_static tests to be executed`
-      );
-    } else {
-      const sszTypeNoUint = replaceUintTypeWithUintBigintType(sszType);
+      throw Error(`Missing SSZ type ${typeName} for fork ${fork}`);
```

Comment:
@jeluard this essentially reverts your PR https://github.com/ChainSafe/lodestar/pull/6776, there is no reason to have if-else as `expect.fail` with throw an error.

Also this code is outside of an `it`-statement whichs means
- it does not make much sense to use `expect.fail`
- it aborts the test run when the error is thrown

I can't really reproduce what you initially stated in the PR
> Do not throw an error when an SSZ type is missing, but rather fail a test.
This let subsequent generated tests run not matter what, ensuring the tests count is consistent.

For me, if an ssz type is missing, it just throws the error and exits

Reply:
We might wanna consider if we wanna abort all static tests if an ssz type is missing or not. But I don't see the point running the ssz static tests if we didn't even define the type, and if someone wants to test only a subset of ssz types (becuase not all have been defined yet) then there is a always the option to pass `-t` flag when running the tests.

Reply:
Removed those changes from this PR, let's discuss in https://github.com/ChainSafe/lodestar/pull/6798